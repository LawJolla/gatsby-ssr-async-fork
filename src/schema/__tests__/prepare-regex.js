const prepareRegex = require(`../prepare-regex`)

describe(`Prepare regex for Sift.js`, () => {
  it(`handles simple regex`, () => {
    expect(prepareRegex(`/blue/`)).toMatchSnapshot()
  })
  it(`handles flags regex`, () => {
    expect(prepareRegex(`/blue/i`)).toMatchSnapshot()
  })
  it(`handles slashes`, () => {
    expect(prepareRegex(`/bl\/ue/i`)).toMatchSnapshot()
  })
})
