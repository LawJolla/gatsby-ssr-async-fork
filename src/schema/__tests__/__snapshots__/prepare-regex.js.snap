// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Prepare regex for Sift.js handles flags regex 1`] = `/blue/i`;

exports[`Prepare regex for Sift.js handles simple regex 1`] = `/blue/`;

exports[`Prepare regex for Sift.js handles slashes 1`] = `/bl\\\\/ue/i`;
